openapi/components/schemas/CurrentProduct.yaml

```
@@ -0,0 +1,51 @@
+type: object
+properties:
+  mp:
+    $ref: ./Marketplace.yaml
+  product_mp_id:
+    description: Id товара на маркетплейсе
+    type: integer
+    format: int64
+  name:
+    description: Имя товара
+    type: string
+  revenue:
+    $ref: ./Revenue.yaml
+  full_price:
+    $ref: ./Price.yaml
+  price:
+    $ref: ./Price.yaml
+  rating:
+    description: Рейтинг товара
+    type: number
+    format: double
+  number_reviews:
```

```suggestions
  reviews:
```
Пусть будет просто reviews